### DISCUSSION TITLE
crf error when trying to access the admin panel

### DISCUSSION THREAD

**Discussion by l0ggik** [evennia/evennia#3170] [Evennia Questions] [answered]
I just set up evennia on my server behind an nginx reverse proxy. Everything works fine but when i tried to login into the admin panel i get a 403 with an "CSRF verification failed. Request aborted." error.

I tried to set
USE_X_FORWARDED_HOST = True
CSRF_TRUSTED_ORIGINS = ['my.url']
but that doesn't help.

Can someone tell me what i have forgotten or why i am getting a csrf error?

Thank you

**Comment by InspectorCaracal** [comment, marked as answer]
You need to make sure the `CSRF_TRUSTED_ORIGINS` is the full URL with the schema, e.g. `['https://my.url']`
